Apps/Sandcastle/gallery/Mars.html

```diff
@@ -0,0 +1,560 @@
+            roverPath.polyline.width = createWidthCallbackProperty(
+              new Cesium.NearFarScalar(0.0, 15.0, 1.0e5, 0.0),
+            );
+
+            toolbarMenuEntries.push({
+              text: entityId,
+              onselect: () => onSelectRover(outRover),
+            });
+
+            return outRover;
+          };
+
+          curiosity = setupRover("Curiosity", 3, curiosity);
+          perseverance = setupRover("Perseverance", 13, perseverance);
+          ingenuity = dataSource.entities.getById("Ingenuity"); // Only for viewing - no data for flight paths
+          theMartianJourney = dataSource.entities.getById("TheMartianJourney");
+          theMartianJourney.polyline.width = createWidthCallbackProperty(
+            new Cesium.NearFarScalar(0.0, 10.0, 1.0e7, 0.0),
+          );
+          theMartianJourney.rectangle.material = new Cesium.ImageMaterialProperty({
+            image: createCanvasAsTexture('Mark Watney\'s Journey in "The Martian"'),
+            transparent: true,
+          });
+
+          toolbarMenuEntries.push({
+            text: '"The Martian" Journey',
+            onselect: () => {
+              reset();
+              viewer.zoomTo(theMartianJourney);
+            },
+          });
+
+          Sandcastle.addToolbarMenu(toolbarMenuEntries);
+        } catch (error) {
+          console.log(`Error loading CZML: ${error}`);
+        }
+
+        // For changing the width of polylines based on distance from the camera
+        function createWidthCallbackProperty(nearFarScalar) {
+          return new Cesium.CallbackProperty(function () {
+            const distance = viewer.camera.positionCartographic.height;
+            let t =
+              (distance - nearFarScalar.near) / (nearFarScalar.far - nearFarScalar.near);
+            t = Cesium.Math.clamp(t, 0.0, 1.0);
+            return Cesium.Math.lerp(nearFarScalar.nearValue, nearFarScalar.farValue, t);
+          }, false);
+        }
+
+        // Converts a Julian date to a Mars Sol number, given a start date / sol number
+        function createJulianDateToSolConverter(startJulianDate, startSol) {
+          return function (julianDate) {
+            const secondsPerSol = 24 * 60 * 60 + 39 * 60 + 35;
+            const differenceInSeconds = Cesium.JulianDate.secondsDifference(
+              julianDate,
+              startJulianDate,
+            );
+            const solNumber = Math.floor(differenceInSeconds / secondsPerSol) + startSol;
+            return `Sol ${solNumber}`;
+          };
+        }
+
+        // Load points of interest from GeoJSON data source
+        try {
+          const dataSource = await Cesium.GeoJsonDataSource.load(
+            "../../SampleData/MarsPointsOfInterest.geojson",
+          );
+          viewer.dataSources.add(dataSource);
+
+          const onSelectLandmark = (landmark) => {
+            reset();
+            scene.camera.flyTo(landmark);
+          };
+
+          const toolbarMenuEntries = [
```

Comment:
If we're keeping 2 lists this should be renamed to make it clear in the code it's a different list than the first time this was declared.